src/defaults.json

```diff
@@ -1,4 +1,16 @@
 {
-  "spacingAliases": {},
-  "base": 24
+  "spacingAliases": {
+    "XS": 0.5,
+    "S/2": 0.5,
+    "S": 1,
+    "M/2": 1,
+    "M": 2,
+    "L/2": 1.5,
+    "L": 3,
+    "XL/2": 2,
+    "XL": 4,
+    "2XL/2": 3,
+    "2XL": 6
+  },
+  "base": 8
```

Comment:
Looking at this aggregated defaults.json, base is confusing.  Maybe we can rename to `baseInPixels`?

Reply:
can solve this in the documentation if needed